ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6276,6 +6276,18 @@ EFHLC:
   implements:
   - OPERATIONAL
 
+FHLC:
+  guid: "4e075523-ef6e-4f3a-917e-1b8ac5b892bd"
```

Comment:
Please remove this line containing the guid. With the new tooling, a guid will be auto-generated upon PR approval.

Reply:
Done